### DISCUSSION TITLE
Can't download index.html files to Mac OS 13

### DISCUSSION THREAD

**Discussion by Steve-Macro** [iterate-ch/cyberduck#14340] [Q&A]
I've recently upgraded to Mac OS 13 and now I can't download index.html files to my computer unless I rename them. Cyberduck seems to be happy that the file has downloaded, e.g. to desktop or downloads folder, but it's not there. I've checked and it's not a hidden file, it's simply not there at all. 

Rename to something else and it downloads fine. I can live with this now I know it happens, it's just a faff having to rename twice, once for download and again on upload.

Anyone know if this is a Cyberduck issue or pure Mac?

**Comment by Cassie205** [comment]
What server software and connection profile do you use? I tested the issue with SFTP and WebDAV but without luck. 
That the download works after renaming the file usually indicates a corrupted file or an issue with the server configuration.
